### PR TITLE
Fix : App is crashing after logging in

### PR DESCRIPTION
**Pull request by Arshilgenius**
Fixes issue #798

### CODE REVIEW

File: app/src/main/java/org/fossasia/susi/ai/adapters/recycleradapters/SearchResultsAdapter.java [hunk 1]
```diff
@@ -54,12 +54,11 @@ public RssViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
     public void onBindViewHolder(final RssViewHolder holder, int position) {
         Datum datum = datumList.get(position);
         if (datum != null) {
-             holder.titleTextView.setText(Html.fromHtml(datum.getTitle()));
-            if(datum.getDescription().isEmpty()) {
-                holder.descriptionTextView.setVisibility(View.GONE);
+            if( datum.getTitle() == null || datum.getTitle().isEmpty()) {
+                holder.titleTextView.setVisibility(View.GONE);
             } else {
-                holder.descriptionTextView.setVisibility(View.VISIBLE);
```
Comment (amitiwary999) [inline app/src/main/java/org/fossasia/susi/ai/adapters/recycleradapters/SearchResultsAdapter.java]: no need to remove descriptiontextview.

**Comment by Arshilgenius** [comment]
@amitiwary999  https://github.com/fossasia/susi_android/pull/807

**Comment by chiragw15** [comment]
Adding check for title does not mean removing check for description view.  We have to display title and description both. 
Add the below code by removing  `holder.titleTextView.setText(Html.fromHtml(datum.getTitle()));` this line and change nothing else.
```
 if( datum.getTitle() == null || datum.getTitle().isEmpty()) {
                holder.titleTextView.setVisibility(View.GONE);
            } else {
                holder.titleTextView.setVisibility(View.VISIBLE);
                holder.titleTextView.setText(Html.fromHtml(datum.getTitle()));
            }
```

**Comment by chiragw15** [comment]
And for updating the code, use rebasing. Use below mentioned commands.
```
git remote add upstream https://github.com/fossasia/susi_android.git
git fetch upstream
git checkout YOUR_BRANCH_NAME
git rebase upstream/development
```
Now do the changes and commit it. You will see your 2 commits on top. Now squash them the using commands I told you in previous PR.

**Comment by Arshilgenius** [comment]
@chiragw15  is it right now?

**Pull request merged**